### PR TITLE
New package: runelite-launcher

### PR DESCRIPTION
**Pull request by mad-s**
[RuneLite](https://runelite.net) is an open-source Old School RuneScape client.

### CODE REVIEW

**Comment by maxice8** [comment]
Please don't introduce binary files, don't vendor licenses.

honestly, avoid vendoring as much as possible

File: srcpkgs/runelite-launcher/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+hostmakedepends="apache-maven-bin"
+depends="virtual?java-runtime"
+short_desc="Open source Old School RuneScape client"
+maintainer="Matteo Signer <matteo.signer@gmail.com>"
+license="BSD-2-Clause"
+homepage="https://runelite.net"
+distfiles="https://github.com/runelite/launcher/archive/${version}.tar.gz"
+checksum=56cefcf906115c1ea4a5cd0556a257672b56a7cf8d8607aba8b1287f5d6ac407
+
+
```
Comment (maxice8) [inline srcpkgs/runelite-launcher/template]: extra line

**Comment by mad-s** [comment]
@maxice8 Could you look at this again, I think it can be merged now that we build it ourselves.

**Comment by maxice8** [comment]
I won't accept while there is a vendored binary file

**Comment by Johnnynator** [comment]
@mad-s It should be ok to add `ImageMagick` to hostmakedepends and use `convert` to get the `.ico` converted to a `png` at build time.

**Comment by mad-s** [comment]
@maxice8 Fixed, I implemented Johnnynator's suggestion to extract the icon.

**Comment by maxice8** [comment]
LGTM, although it would be nice (doesn't block inclusion of the package) that issues were raised upstream so we don't have to vendor these pieces (convert the .ico to .png and the .desktop file)

**Pull request merged**